src/main/java/bio/terra/service/iam/sam/SamIam.java

```diff
@@ -157,6 +159,29 @@ private boolean isAuthorizedInner(
     }
   }
 
+  @Override
+  public Map<UUID, Set<IamRole>> listAuthorizedResourcesAndRoles(
```

Comment:
This code is the same as `listAuthorizedResourcesAndRoles()`, except that version discards the roles information. Instead of adding a new API, I could change all the upstream code to use the new version. The main reason I didn't change it right away was because we have code that mocks this API at the SamIam layer (instead of the layer above, in `IamService`). I'm going to look at removing the duplication before merging this up.